<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_next">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt; Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_operating_temp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_vgs">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_power_max">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_operating_temp">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Operating Temperature</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_max_power">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Power - Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Transistor Type</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_tfmr_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vgs">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vgs</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_vce_sat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_gain">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_vce_breakdown">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vce_sat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vce Saturation</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vce_breakdown">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vce Breakdown</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_collector_current">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current - Collector (Ic)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_collector_current">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_gain">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>DC Current Gain</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_drive_voltage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_drain_current">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current - Drain (Id)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_drain_current">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_vdss">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Vdss</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_vdss">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_drive_voltage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Drive Voltage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="box_rds_on">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rds_on">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Rds On</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_ohms">
   <property name="geometry">
    <rect>
     <x>408</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Î©</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
